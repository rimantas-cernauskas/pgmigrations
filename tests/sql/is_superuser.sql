-- Test function pgmigrations.is_superuser()
\i tests/sql/setup/init.sql

BEGIN;
SELECT plan(2);

SET ROLE __pgmigrations_testuser;
SELECT is(
     pgmigrations.is_superuser()
    , FALSE
    , 'should get FALSE for non superuser'
);

SET ROLE __pgmigrations_superuser;
SELECT is(
     pgmigrations.is_superuser()
    ,TRUE
    ,'should get TRUE for superuser'
);

SELECT * FROM finish();
ROLLBACK;
