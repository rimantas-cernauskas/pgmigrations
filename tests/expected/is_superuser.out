-- Test function pgmigrations.is_superuser()
\i tests/sql/setup/init.sql
\set ECHO errors
1..2
ok 1 - should get FALSE for non superuser
ok 2 - should get TRUE for superuser
